### DISCUSSION TITLE
Como puedo jugar Local con mis amigos?

### DISCUSSION THREAD

**Discussion by Roanvil** [mupen64plus-ae/mupen64plus-ae#1133] [Q&A]
Tengo una pregunta, yo tengo el Super Smash Bros y el Super Mario 64 Split Screen pero cuando quiero jugar por WiFi o por zona WiFi, el NetPlay no me da mi Código, dice que no se pudo conseguir, y a mis amigos les sale lo mismo. Tenemos el Mupen64Plus FZ Pro

Que hago?

**Comment by fzurita** [comment]
Mira a esta respuesta:
https://github.com/mupen64plus-ae/mupen64plus-ae/discussions/1133

**Reply by Roanvil** [reply]
> Mira esta respuesta: #1133

?
